docs/source/mcmc_samplers/monomial_gamma_hmc_mcmc.rst

```diff
@@ -0,0 +1,7 @@
+***********************
+Monomial-Gamma HMC MCMC
+***********************
+
+.. currentmodule:: pints
+
+.. autoclass:: MonomialGammaHMCMCMC
```

Comment:
Is there any way we can avoid this name?
(As an MC myself I should be pleased, really)

Reply:
I note we've called HMC "Hamiltonian MCMC" in some places

Reply:
Good point -- too many MCs ruin the music (forgive the pun). I've changed it throughout to monomial gamma hamiltonian MCMC